lib/egl/egl.hpp

```diff
@@ -42,9 +41,12 @@ class EglWindow : public GLWindow
       EGLConfig eglCfg{};
 #endif
 #ifdef GLVIS_USE_CGL
-      GLuint buf_frame {}, buf_color {}, buf_depth {};
-      CGLPixelFormatObj pixFmt {};
-      CGLContextObj ctx{};
+      GLuint buf_frame, buf_color, buf_depth;
+      CGLPixelFormatObj pix;
```

Comment:
I think it would not hurt to initialize these to be sure we have a complete handle, but it is true it is implementation dependent 🤔

Reply:
done in https://github.com/GLVis/glvis/pull/345/commits/e873d0c5c2524b4b1fb6bdc0669970a323cbfd5a